src/app/Containers/Trailer/index.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { getMovie } from '../Movie/duck/operations';
+import { SUCCESS } from '../../../common/constants';
+import { Trailer } from '../../Components';
+
+const TrailerContainer = ({ match }) => {
+  const {
+    params: { movieId },
+  } = match;
+
+  const dispatch = useDispatch();
+  const [title, setTitle] = useState('');
+  const [url, setUrl] = useState('');
+  const { state, content } = useSelector((state) => state.movie);
+
+  useEffect(() => {
+    dispatch(getMovie(movieId));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    if (state === SUCCESS) {
+      setTitle(content.title);
+      setUrl(content.trailer);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [content]);
```

Comment:
We don't really need to have the _title_ and _url_ on the state if we already have it on the redux store. 
We should directly use the values from the store

Reply:
And after removing that state logic, this component doesn't really makes sense, don't you think?

Reply:
Isn't the component necessary because it has the responsibility to dispatch the action that retrieves the movie?